ci/vars-nightly_bundles.yml

```diff
@@ -0,0 +1,7 @@
+---
+# from: https://github.com/infrawatch/service-telemetry-operator/pull/437
+# ansible-playbook -e __service_telemetry_storage_ephemeral_enabled=true -e __local_build_enabled=false -e __deploy_from_bundles_enabled=true -e __service_telemetry_bundle_image_path=quay.io/infrawatch-operators/service-telemetry-operator-bundle:nightly-head -e __smart_gateway_bundle_image_path=quay.io/infrawatch-operators/smart-gateway-operator-bundle:nightly-head --skip-tags bundle_registry_tls_ca --skip-tags bundle_registry_auth build/run-ci.yaml
```

Comment:
We might have to start considering tests that span `stable-1.5` and `master` since the nightly builds are results from different branches and thus different code bases. As we move towards STF 2.0 (and even STF 1.5.4) we'll have different code bases that could really both use separate test paths.

Reply:
(not for now, I'm happy to just see this start landing against `master` for now, but this code will eventually make its way into a stable-1.5 branch at some point, and we'll need to make some adjustments when that happens in a week or two)

Reply:
This nightly check might well just run periodically, once a week or so.
To get tests spanning multiple branches, we need to add the job to that branch.

I think we also need a few more updates to get it working, but I'm focussing on getting the jobs on master first

Reply:
Yep makes sense. Just noting that once we merge this to stable-1.5 as part of STF 1.5.3 then these will technically be the wrong tags.